src/tls.c

```diff
@@ -3113,11 +3126,29 @@ static int TLSX_CSR_Parse(WOLFSSL* ssl, byte* input, word16 length,
         if (ret != WOLFSSL_SUCCESS)
             return ret; /* throw error */
 
-    #if defined(WOLFSSL_TLS13) && !defined(NO_WOLFSSL_SERVER)
```

Comment:
This code isn't server specific. Client needs to be able to verify certificate status, too. See: https://www.wolfssl.com/forums/topic1639-example-of-a-client-that-doesnt-attempt-to-contact-ocsp-responder.html